Dockerfile

```diff
@@ -7,12 +7,20 @@ RUN mkdir -p $GOBIN
 
 COPY . $GOPATH/src/github.com/grafana/grafana
 
-RUN yum install -y golang make git && \
-    cd $GOPATH/src/github.com/grafana/grafana && \
-    go run build.go build && \
-    cp $GOPATH/src/github.com/grafana/grafana/bin/linux-$(go env GOARCH)/grafana-server /usr/bin/ && \
-    yum autoremove -y golang make git && yum clean all && \
-    rm -rf .git
+#RUN yum install -y make git
+RUN cd $GOPATH/src/github.com/grafana/grafana && go run build.go build
+RUN cp $GOPATH/src/github.com/grafana/grafana/bin/linux-$(go env GOARCH)/grafana-server /usr/bin/
+RUN rm -rf $GOPATH/src/github.com/grafana/grafana/.git
+
+FROM openshift/origin-base
+
+ENV GOPATH="/go"
+ENV GOBIN="${GOPATH}/bin"
+ENV PATH="${GOBIN}:${PATH}"
```

Comment:
do you really want `/go/bin` in your path when you're not putting anything there?